dmapiclient/data.py

```diff
@@ -422,6 +422,15 @@ def is_email_address_with_valid_buyer_domain(self, email_address):
             "/users/check-buyer-email", params={'email_address': email_address}
         )['valid']
 
+    def create_buyer_email_domain(self, buyer_email_domain, user):
+        return self._post_with_updated_by(
+            "/buyer-email-domain",
+            data={
+                "buyerEmailDomains": {"domainName": buyer_email_domain}
```

Comment:
Might we ever want to do a bulk update?  Probably not I guess, they only come in one at a time...

Reply:
Out of scope 😸